### PR TITLE
New package: libsakura

### PR DESCRIPTION
**Pull request by pkgw**
An image scaling library.

### CODE REVIEW

**Comment by pkgw** [comment]
Ready for review and merging if all is well.

**Comment by pkgw** [comment]
Ping @ocefpaf @jakirkham etc?

File: recipes/libsakura/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+about:
+  home: http://alma.mtk.nao.ac.jp/j/
+  license: GPLv3
+  license_file: COPYING
+  summary: 'C++ library for fast image manipulations'
+
+extra:
+  recipe-maintainers:
+    pkgw
```
Comment (jakirkham) [inline recipes/libsakura/meta.yaml]: The maintainers should be a list. So this should be prefixed with a `- `. Please submit a PR to the feedstock and ping core to merge it. Thanks.

Comment (jakirkham) [inline recipes/libsakura/meta.yaml]: Resolved in PR ( https://github.com/conda-forge/libsakura-feedstock/pull/1 ).

Comment (jakirkham) [inline recipes/libsakura/meta.yaml]: Also fixed in `master` here too just in case.

xref: https://github.com/conda-forge/staged-recipes/pull/2530

**Pull request merged**